code/modules/organs/blood.dm

```diff
@@ -125,11 +125,23 @@ var/const/BLOOD_VOLUME_SURVIVE = 40
 
 		//Bleeding out
 		var/blood_max = 0
+		var/blood_loss_divisor = 30	//lower factor = more blood loss
+		//This 30 is the "baseline" of a cut in the "vital" regions (head and torso).
 		for(var/obj/item/organ/external/temp in organs)
 			if(!(temp.status & ORGAN_BLEEDING) || (temp.robotic >= ORGAN_ROBOT))
 				continue
 			for(var/datum/wound/W in temp.wounds)
 				if(W.bleeding())
+					if(W.damage_type == PIERCE) //gunshots and spear stabs bleed more
+						blood_loss_divisor -= 5
+					if(W.damage_type == BRUISE) //bruises bleed less
+						blood_loss_divisor += 5
+					//the farther you get from those vital regions, the less you bleed
+					//depending on how dangerous bleeding turns out to be, it might be better to only apply the reduction to hands and feet
+					if((temp.organ_tag == ARM_LEFT) || (temp.organ_tag == ARM_RIGHT) || (temp.organ_tag == LEG_LEFT) || (temp.organ_tag == LEG_RIGHT))
```

Comment:
Leaving bleeding partially up to RNG was one of the concepts I spitballed around a bit before doing this, but I decided I'd rather not do that after talking with a couple people about it. Leaving lethality up to RNG isn't very fun, and ends up giving Medbay something to tussle with that they can't actually measure in any way.